### PR TITLE
transitive const

### PR DESCRIPTION
**Pull request by wilzbach**

### CODE REVIEW

**Comment by stonemaster** [comment]
@wilzbach merge conflicts

**Comment by stonemaster** [comment]
This somehow conflicts with #168 because here are the changes from #168 too.

**Comment by wilzbach** [comment]
> @wilzbach merge conflicts

Rebased (hopefully now correctly).

File: public/content/en/basics.md [hunk 1]
```diff
@@ -197,14 +197,22 @@ pointer can be created from either a *mutable* or
 is `const` for your current scope, but someone
 else might modify it in future. Just with an `immutable`
 you will be sure that an object's value will never
-change.
+change. It is common for API's to accept `const` types
```
Comment (stonemaster) [inline public/content/en/basics.md]: APIs because its plural

File: public/content/en/basics.md [hunk 2]
```diff
@@ -197,14 +197,22 @@ pointer can be created from either a *mutable* or
...
 
     immutable a = 10;
     int b = 5;
     const int* pa = &a;
     const int* pb = &b;
     *pa = 7; // disallowed
 
+Both `immutable` and `const` are transitive which ensures that once `const` is applied to a type, it applies recursively to every sub-component of that type.
+    
+### In-depth
```
Comment (stonemaster) [inline public/content/en/basics.md]: That doesn't look right. There are two in-depth sections after another

**Comment by wilzbach** [comment]
Adressed & rebased.

**Pull request merged**